patches/chrome-elevation_service-service_main.cc.patch

```diff
@@ -0,0 +1,42 @@
+diff --git a/chrome/elevation_service/service_main.cc b/chrome/elevation_service/service_main.cc
+index ef400534ef15cc8b7aeab32f0e71c664e7e5c0d3..83887db6257192a6f9d7c82c0d29af2cb522892f 100644
+--- a/chrome/elevation_service/service_main.cc
++++ b/chrome/elevation_service/service_main.cc
+@@ -32,7 +32,7 @@ namespace {
+ // Command line switch "--console" runs the service interactively for
+ // debugging purposes.
+ constexpr char kConsoleSwitchName[] = "console";
+-constexpr wchar_t kWindowsServiceName[] = L"ChromeElevationService";
++std::wstring kWindowsServiceName;
```

Comment:
This patch is ugly - I don't know why Chrome hard codes the entry. With the name set like this, it would only match the stable Chrome. 🤷

Reply:
If reviewers approve of this patch, please consider maybe something like:

```c++
const std::string& GetWindowsServiceName() {
  static base::NoDestructor<std::string> name = install_static::GetElevationServiceName();
  return *name;
}
```

To avoid any potential issues during termination.

Reply:
this can be implemented via `chromium_src` override.

https://gist.github.com/goodov/b41cbc039463c674b180764923c3cb08

`kWindowsServiceName` removal is intentional to ensure we won't start using incorrect value after some rebase.

Reply:
I don't see this patch is improved. @bsclifton have you looked into this?https://gist.github.com/goodov/b41cbc039463c674b180764923c3cb08

you can replace this patch with `chromium_src` override that:
adds this into header:
```
static const wchar_t* GetWindowsServiceName();
const wchar_t* kWindowsServiceName = GetWindowsServiceName();
```

adds this into cc:
```
// Remove constexpr from `dispatch_table` declaration.
#define SERVICE_TABLE_ENTRY                               \
  SERVICE_TABLE_ENTRY kUnusedTable [[maybe_unused]] = {}; \
  static const SERVICE_TABLE_ENTRY

// static
const wchar_t* ServiceMain::GetWindowsServiceName() {
  static const base::NoDestructor<std::wstring> kServiceName(
      install_static::GetElevationServiceName());
  return kServiceName->c_str();
}
```

removes this from upstream cc:
```
constexpr wchar_t kWindowsServiceName[] = L"ChromeElevationService";
```

Reply:
Checking this out now! Thanks 🙂👍

Reply:
Fixed!